packages/client/package.json

```diff
@@ -132,7 +133,7 @@
     "jest-styled-components": "^7.1.1",
     "jsdom": "^21.1.1",
     "openfin-cli": "^4.0.0",
-    "playwright": "^1.29.1",
+    "playwright": "^1.37.1",
```

Comment:
Thanks for highlighting that. Your suggestion is to set it to the exact version like `1.37.1` ? Should we set it to `~1.37.1` to allow getting any patches that we may not be aware of ?

Reply:
nah, don't think so ... the versions are fixed in the lock file anyway, so we don't automatically get patch (or any) updates .. this would just be if we selected to upgrade a dependency, with the original version spec untouched .. I just don't think that happens .. so no point in being "loose" (IMO) unless you specifically want to communicate that we are prepared to update at any point, but only within a specific range .. those are rare cases I think

Reply:
perfect, I did set them to be explicit 👍